applications.html

```diff
@@ -24,10 +24,21 @@ <h3>APPLICATIONS</h3>
           <a href="#nature">Nature</a>
         </li>
 
-        <li>
-
-         <a href="#art">Art</a>
-        </li>
+          <li>
+
+            <a href="#art">Art</a>
+          </li>
+          <li>
```

Comment:
just add here
<li>
          <a href="#architec">ARCHITECTURE</a>
        </li>

Reply:
Can You review it again. I changed Arts to Architecture